spec/acceptance/peadm_spec/plans/test_migration.pp

```diff
@@ -10,6 +11,7 @@
   $primary_target = $primary_host ? { '' => undef, default => peadm::get_targets($primary_host, 1) }
   $new_primary_target = $new_primary_host ? { '' => undef, default => peadm::get_targets($new_primary_host, 1) }
   $new_replica_target = $new_replica_host ? { '' => undef, default => peadm::get_targets($new_replica_host, 1) }
+  $primary_postgresql_target = $primary_postgresql_host ? { '' => undef, default => peadm::get_targets($primary_postgresql_host, 1) }
```

Comment:
we can remove this line - we already pass in 'new_primary_postgresql_target' so we dont need to pass in this 'primary_postgresql_target' parameter